BabelfishFeatures.cfg

```diff
@@ -14,16 +14,20 @@
 
 # this must always be the first section:
 [Babelfish for T-SQL]
-valid_versions=1.0.0, 1.1.0, 1.2.0, 1.2.1, 1.3.0, 2.1.0 # only Babelfish version numbers listed here can be referenced below
+# only Babelfish version numbers listed here can be referenced the rules below:
+valid_versions=1.0.0, 1.1.0, 1.2.0, 1.2.1, 1.3.0, 1.3.1, 2.1.0, 2.1.1 
+# 1.2.1 is a bugfix release for 1.2.0 (no new T-SQL features supported)
+# 1.3.1 is a bugfix release for 1.3.0 (no new T-SQL features supported)
+# 2.1.1 is a bugfix release for 2.1.0 (no new T-SQL features supported)
 file_format=1                           # version number for format of this .cfg file. This is not expected to change much 
-file_timestamp=28-Jun-2022              # identifies the version of this file, together with the latest Babelfish version supported
+file_timestamp=Jul-2022                 # identifies the version of this file, together with the latest Babelfish version supported

```

Comment:
Do we need to include the day as well? I noticed the prior release had this information.

Reply:
We don't need the day, it is optional. It was only included on the previous version since there were two release in one month.